### DISCUSSION TITLE
Checkin in RE 4725 nicht möglich

### DISCUSSION THREAD

**Discussion by Felix26** [Traewelling/traewelling#2593] [Questions / Support]
Nach RE 4725 kann man weder von Offenburg, noch von Donaueschingen und Villingen einchecken. Hat jemand eine Ahnung, woran das liegt? Danke für eventuelle Hilfe!
![IMG-20240525-WA0001](https://github.com/Traewelling/traewelling/assets/26878188/5dcc23ef-80a5-441a-90df-1292a8d5a0be)

**Comment by pascalwittler** [comment]
Gerade habe ich versucht, dein Problem im Nachhinein für den 25.05.2024 nachzustellen und konnte erfolgreich von Donaueschingen nach Konstanz einchecken [1].

Da ich (noch? :eyes: ) nicht zum Entwicklerteam von Träwelling gehöre, kann ich nur aus Erfahrung sagen, dass die Fehlermeldung bei mir sehr vereinzelt auch bei unterschiedlichen Reisen, v.a. im Nahverkehr, erscheint und beim zweiten oder dritten unmittelbaren Versuch, nochmal auf den selben Zug aus der Liste zu klicken, funktioniert der Check-In dann.

[1] Erfolgreicher Check-in in RE 4725 am 25.05.2024

![1](https://github.com/Traewelling/traewelling/assets/20669380/ebe052ce-7f20-4702-a2c2-fd265d02df94)

**Reply by MrKrisKrisu** [reply]
Der Fehler "Start ID nicht in den Zwischenstopps" ist mir bislang in dem Umfang leider auch ein Rätsel. Es passiert gelegentlich. Auch bei mir. Aber sobald ich einen Fall debuggen möchte kann ich es nicht mehr nachstellen und alles funktioniert. 😐
